boards/acd52832/Makefile.dep

```diff
@@ -0,0 +1,13 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
```

Comment:
This file seems to be a copy of the one in ```boards/nrf52dk```. Please deduplicate.

Reply:
I assume it is. What kind of deduplication do you have in mind, something like a board_common folder?

Reply:
> What kind of deduplication do you have in mind, something like a board_common folder?

Yes.
In this case, I'd suggest a folder called "nrf5x-common". The non-common nrf5x Makefile.dep would contain a line that just include the common one.

Reply:
Technically I understand your proposal to avoid duplication. On the other hand this is board specific and the board here is not an NRF Nordic board but the [aconno](http://aconno.de/acd52832/). I fear this might get un-intuitive. For example: I find it quite un-intuitive that the waspmote-pro depends on `arduino-atmega-common`. Did I change your mind :-)? If not, I still think we should introduce a common folder in a separate PR, right?

Reply:
So you think we should wait for something like #8044 right?

Reply:
Yes... Maybe #8102!

Reply:
Sure but before I start reviewing #8102 I'd like to see #8058 merged